index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My CV</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="https://www.flaticon.com/svg/static/icons/svg/909/909212.svg">
+</head>
+<body>
+    <main>
+        <h1>Junior Front-End Developer Resume</h1>
+        <section>
+            <ul>
+                <li><span>ANDREY PAVLOV</span></li>
+                <li><span>9000andrey@gmail.ru</span></li>
+                <li><span>+375295080289</span></li>
+                <li><span>github.com/AndreiPavlov135</span></li>
+                <li>
+                    <p>Objective</p><span>My goal is to complete the course, get the skill of a
+                        developer, gain
+                        knowledge from
+                        different
+                        places,
+                        because when I come to work, no one will teach me there. I can learn quickly.</span>
+                </li>
+                <li>
+                    <p>Skills</p><span>At this stage I know the basic HTML, CSS, JS.</span>
+                </li>
+                <li>
+                    <p>Code example:</p>
+                    <div class="codexample">
+                        <span class="codexampl">
```

Comment:
Don't use the similar name of the class for different elements: .codexample and .codexampl. It can provide different syntaxis errors in the future and makes code unreadable